frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -786,6 +806,17 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
                   title={`${status.phase}: ${status.message}`}
                 />
               )}
+              {initializationResource && (
+                <Alert
+                  isInline
+                  className="co-alert"
+                  variant="warning"
+                  title="Custom resource required"
+                >
+                  <p>Create the required custom resource to be able to use this operator.</p>
```

Comment:
This is more of a design/content issue, but the wording of the title and content sound a bit weird to me. The phrase "to be able to" should just be "to", and IMO we should use the cr kind in the title and content. For example (made up a variable for the interpolated kind):
```suggestion
                  title={`${requiredCR.kind} Required`}
                >
                  <p>Create a ${requiredCR.kind} instance to use this operator.</p>
```

Reply:
@tlwu2013 @itsptk WDYT?

Reply:
Agreed. If we have access to the resource name we may as well use it. Thanks!